eng/pipelines/templates/steps/analyze.yml

```diff
@@ -23,4 +23,4 @@ steps:
       arguments: >
         -Toolchain '$(Toolchain)'
         -PackageInfoPath '$(Build.ArtifactStagingDirectory)/PackageInfo'
-        -SkipPackageAnalysis:$$(NoPackagesChanged)
+        -SkipPackageAnalysis:('$(NoPackagesChanged)' -eq 'true')
```

Comment:
Don't you still need the `$` in front given PS is going to interpret this?

Reply:
Nevermind I see PS would be interpreting the boolean expression so it shoulld have the correct value.